modules/oe_paragraphs_illustrations_lists/src/EventSubscriber/OptionsEventSubscriber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_paragraphs_illustrations_lists\EventSubscriber;
+
+use Drupal\oe_paragraphs_illustrations_lists\Event\ColumnOptionsEvent;
+use Drupal\oe_paragraphs_illustrations_lists\Event\RatioOptionsEvent;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Provides options for the column and ratio fields.
+ */
+class OptionsEventSubscriber implements EventSubscriberInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      ColumnOptionsEvent::class => 'getColumnOptions',
+      RatioOptionsEvent::class => 'getRatioOptions',
```

Comment:
It will work, but curious why you decided to use a fully qualified class name instead of, as usually, declared in event class constant with the event name.

Reply:
We had a conversation about it with Fra:
```

Francesco Sardara  12:09 PM
new way of launching events is by launching the event class itself





12:09
so putting outside the event name is outdated for me
12:10
you can see it if you have locally like 9.3
12:10
follow the dispatch
12:10
unfortunately we have to support 8.9 otherwise you pass already the event to the dispatcher

Evgenii Nikitin  12:12 PM
yeah, I saw it but Drupal 10 still uses event naming.
ok, will use class as a event name

Francesco Sardara  12:15 PM
dunno about 10 currernt status but
12:15
@trigger_error('Calling the Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch() method with a string event name as the first argument is deprecated in drupal:9.1.0, an Event object will be required instead in drupal:10.0.0. See https://www.drupal.org/node/3154407', E_USER_DEPRECATED);
12:15
they swap it at least
12:16
+ they have some code that you can pass the event name or just object
12:17
in most cases i think just object should be fine, unless there are multiple events with same class… who knows
```